float8_experimental/float8_linear_utils.py

```diff
@@ -322,3 +332,47 @@ def inner_func():
     for child in fp8_layers:
         # Set a flag to signal amaxes/scales are ready
         child.amax_and_scale_synced = True
+
+
+def precompute_float8_weights(module: nn.Module) -> None:
```

Comment:
maybe these kind of utils warrant their own file?

Reply:
yeah, we had a distributed utils file that we can add back